tests/Jellyfin.Model.Tests/Dlna/StreamBuilderTests.cs

```diff
@@ -162,7 +162,7 @@ public class StreamBuilderTests
         [InlineData("Tizen4-4K-5.1", "mkv-vp9-aac-srt-2600k", PlayMethod.DirectPlay)]
         [InlineData("Tizen4-4K-5.1", "mkv-vp9-ac3-srt-2600k", PlayMethod.DirectPlay)]
         [InlineData("Tizen4-4K-5.1", "mkv-vp9-vorbis-vtt-2600k", PlayMethod.DirectPlay)]
-        public async Task BuildVideoItemSimple(string deviceName, string mediaSource, PlayMethod? playMethod, TranscodeReason why = (TranscodeReason)0, string transcodeMode = "DirectStream", string transcodeProtocol = "")
+        public async Task BuildVideoItemSimple(string deviceName, string mediaSource, PlayMethod? playMethod, TranscodeReason why = 0, string transcodeMode = "DirectStream", string transcodeProtocol = "")
```

Comment:
Maybe `default` would be better than `0`?

Reply:
@Bond-009, There is a chance to change the behavior of the code in this case as enum starts from 1

Reply:
`default` for enums should always be `0` AFAIK?

Reply:
Except when explicitly specified :)
You can check the source code of the enum and make sure

Reply:
Please show me how/where, I can only find this:
> The default value of an enumeration type E is the value produced by expression (E)0, even if zero doesn't have the corresponding enum member.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum

Reply:
You are absolutely right. I thought it sets the first enum value (1), but it sets 0 even if there is such a value.

Reply:
So let's use default in this case

Reply:
@Bond-009, done